build/build.yml

```diff
@@ -17,14 +17,14 @@ steps:
   displayName: 'dotnet build $(buildConfiguration)'
   inputs:
     command: build
-    arguments: '--configuration $(buildConfiguration) /p:AssemblyVersion="$(assemblySemVer)" /p:FileVersion="$(assemblySemFileVer)" /p:InformationalVersion="$(informationalVersion)" /warnaserror'
+    arguments: '--configuration $(buildConfiguration) /p:AssemblyVersion="$(assemblySemVer)" /p:FileVersion="$(assemblySemFileVer)" /p:InformationalVersion="$(informationalVersion)" /p:ContinuousIntegrationBuild=true /warnaserror'
```

Comment:
This [flag](https://github.com/dotnet/sourcelink/blob/master/docs/README.md#continuousintegrationbuild) sets determinism-related flags that would only apply on a CI server, like deterministic source paths, etc.